models/intermediate/int_jira.yml

```diff
@@ -576,17 +599,17 @@ models:
       - name: issue_day_id
         description: Unique ID hashed on `valid_starting_on` and `issue_field_history`.
         tests:
-          - unique
```

Comment:
Same request here to revert and include source_relation in the hash

Reply:
Also changed `issue_field_history` to `issue_id`